.github/workflows/maven.yml

```
@@ -71,25 +71,37 @@ jobs:
   test_coatjava:
     needs: [ build ]
     strategy:
-      fail-fast: true
+      fail-fast: false
```

consider reverting this before merging (currently `false` to see which tests fail)
```suggestion
      fail-fast: true
```